cmd/ec2.go

```diff
@@ -15,21 +19,52 @@ var ec2Cmd = &cobra.Command{
 	Short: "Print ec2 related information",
 	Long: `For example,
 		awsctl get ec2`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("ec2 called")
-	},
+	Run: ec2FunDescribe,
 }
 
-func init() {
-	getCmd.AddCommand(ec2Cmd)
+type EC2DesAPI interface {
+	DescribeInstances(ctx context.Context,
+		params *ec2.DescribeInstancesInput,
+		optFns ...func(*ec2.Options)) (*ec2.DescribeInstancesOutput, error)
+}
+
+// Returns info about ec2 instance
+func GetEc2Info(ctx context.Context, input *ec2.DescribeInstancesInput, api EC2DesAPI) (*ec2.DescribeInstancesOutput, error) {
+
+	return api.DescribeInstances(ctx, input)
+
+}
+
+func ec2FunDescribe(cmd *cobra.Command, args []string) {
```

Comment:
I would rename it to getEC2command as this function will execute on `awsctl get ec2`

Reply:
Okay